lib/ModuleReason.js

```diff
@@ -4,9 +4,45 @@
 */
 "use strict";
 
-module.exports = class ModuleReason {
+const util = require("util");
+
+class ModuleReason {
 	constructor(module, dependency) {
 		this.module = module;
 		this.dependency = dependency;
+		this._chunks = null;
```

Comment:
originally I was going to do `this._chunks = new Set(this.module._chunks);` but believe it or not, 1 tests in the test suite breaks (with a missing module) if I don't do the lazy caching below.